checks.d/docker_daemon.py

```diff
@@ -643,6 +656,39 @@ def _format_events(self, aggregated_events, containers_by_id):
 
         return events
 
+    def _report_disk_stats(self):
+        """Report metrics about the volume space usage"""
+        stats = {
+            'used': None,
+            'total': None,
+            'free': None
+        }
+        info = self.client.info()
+        driver_status = info.get('DriverStatus', [])
+        for metric in driver_status:
+            if metric[0] == 'Data Space Used':
+                stats['used'] = metric[1]
+            elif metric[0] == 'Data Space Total':
+                stats['total'] = metric[1]
+            elif metric[0] == 'Data Space Available':
+                stats['free'] = metric[1]
+        stats = self._format_disk_metrics(stats)
+        tags = self._get_tags()
+        for name, val in stats.iteritems():
+            if val:
+                self.gauge(DISK_METRICS_PREFIX.format(name), val, tags)
+
+    def _format_disk_metrics(self, metrics):
+        """Cast the disk stats to float and convert them to bytes"""
+        for name, raw_val in metrics.iteritems():
+            if raw_val:
+                val, unit = raw_val.split(' ')
+                # by default some are uppercased others lowercased. That's error prone.
+                unit = unit.lower()
+                val = int(float(val) * UNIT_MAP.get(unit, 1))
```

Comment:
yeah that makes sense. I'll add `tb` and error out with a log message when the unit's not recognized.